### DISCUSSION TITLE
Login Page also with react

### DISCUSSION THREAD

**Discussion by T0biii** [freifunkMUC/wg-access-server#632] [Ideas]
i was wondering if the Login Page can also be styled like the content under [website/](https://github.com/freifunkMUC/wg-access-server/tree/master/website?rgh-link-date=2024-03-28T13%3A53%3A11Z)

so it can benefit of the e.g. better darkmode/responsive design

**Comment by GoliathLabs** [comment]
Would need a PR and partial rewrite of the logic.
